conf/app.routes

```diff
@@ -1,3 +1,13 @@
 # microservice specific routes
 
-POST        /id         uk.gov.hmrc.apisubscriptionfields.controller.ApiSubscriptionFieldsController.id
+# TODO add endpoints for field definitions CRUD operations
+
```

Comment:
Do we really need both of these two routes?
```
GET         /id/:appId/:apiContext/:apiVersion
GET         /application/:appId/context/:apiContext/version/:apiVersion  
```
I would keep only one. 
I think they do exactly the same thing. 
The first one returns just the `id`. 
The second returns both `id` and `fields`.
I would keep only one that returns everything in the payload.

Reply:
If the subscription-fields-id header injection code is OK receiving the larger payload, then the /id/... endpoint can go.

Reply:
It is just an extra field in the payload. 
If I need just the `id`, I will parse only the `id` in the request. I won't consider the fields.

Reply:
id endpoint removed

Reply:
Ok.
Remember to commit and push all your changes to this remote branch.
Then we can validate the new code